inst/rmarkdown/templates/nhsr-presentation/skeleton/skeleton.Rmd

```diff
@@ -5,7 +5,22 @@ author: "Author Name"
 institute: "NHS Organisation"
 date: "Presentation Date"
 output:
-  nhsrtheme::nhsr_presentation: default
+  xaringan::moon_reader:
+    css: 
+      - default
+      - css/nhsr.css
+      - css/nhsr-fonts.css
+    lib_dir: libs
+    seal: false
+    self_contained: TRUE
```

Comment:
like above, I throw an error. But this one is probably not quite as true as with seal, I just think it makes life easier to self contain everything... might be worth relaxing that rule in nhsr_presentation()

Reply:
I agree, easier to self contain everything.